vultr.php

```diff
@@ -901,10 +901,10 @@ public function getClientAddFields($package, $vars = null)
         Loader::loadHelpers($this, ['Html']);
 
         // Get the available templates
-        $templates = $this->getTemplates($this->module_row, $package);
+        $templates = $this->getTemplates($this->getModuleRow($package->module_row), $package);
```

Comment:
Instead of fetching the module row twice, you should assign it to a variable and use it like ::getAdminAddFields and ::getAdminEditFields